mode-apis-base/src/wsConnection.ts

```diff
@@ -1,90 +1,199 @@
+/* eslint-disable no-param-reassign */
 /* eslint-disable no-console */
 /* eslint-disable max-classes-per-file */
-export interface ModeWebSocketEvent {
+
+
+export interface WebSocketEvent {
     readonly eventType: string;
-    readonly eventData: any;
+    readonly eventData: unknown;
```

Comment:
Purpose of this change is to force a developer to narrow type before assigning to something else?

Reply:
yes, it can be anything depending on the Gateway's implementation.